From 9291ba9c15e6e3d203a971424af50e59d3411506 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Mon, 3 Aug 2020 12:30:36 +0800
Subject: [PATCH] Test case4 for Change guid value in Protocols

Description: Change guid value in Protocols section (to gEfiLoadPeImageProtocolGuid = { 0x5CB5C546, 0x60D5, 0x45FE, { 0x88, 0x3C, 0x45, 0x37, 0x78, 0xDD, 0x4B, 0x3F }}) in MdeModulePkg/MdeModulePkg.dec.

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 MdeModulePkg/MdeModulePkg.dec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MdeModulePkg/MdeModulePkg.dec b/MdeModulePkg/MdeModulePkg.dec
index 843e963ad3..cce15e2218 100644
--- a/MdeModulePkg/MdeModulePkg.dec
+++ b/MdeModulePkg/MdeModulePkg.dec
@@ -471,7 +471,7 @@
   #  Include/Protocol/LoadPe32Image.h
   #  This protocol is deprecated. Native EDKII module should NOT use this protocol to load/unload image.
   #  If developer need implement such functionality, they should use BasePeCoffLib.
-  gEfiLoadPeImageProtocolGuid    = { 0x5CB5C776, 0x60D5, 0x45EE, { 0x88, 0x3C, 0x45, 0x27, 0x08, 0xCD, 0x74, 0x3F }}
+  gEfiLoadPeImageProtocolGuid    = { 0x5CB5C546, 0x60D5, 0x45FE, { 0x88, 0x3C, 0x45, 0x37, 0x78, 0xDD, 0x4B, 0x3F }}
 
   ## Print protocols define basic print functions to print the format unicode and ascii string.
   # Include/Protocol/Print2.h
-- 
2.27.0.windows.1

